apps/website/.vuepress/theme/global-components/ReleaseGroup.vue

```diff
@@ -4,6 +4,7 @@
     <table class="table">
       <tr>
         <th class="left" style="width: 100px;">Scope</th>
+        <th class="left" style="width: 100px;">Project</th>
```

Comment:
This should be a rem value (`5rem` iirc)

Px values are not accessible

Reply:
You are right, that's what I get for copy paste 😔.